CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
```

Comment:
"before the end of the layer"? Not sure what is meant by this in the context of a go project. Makes sense for a Dockerfile, but don't think this is relevant for Go. Probably something about ensuring that you've vendored your dependencies into the project using `dep ensure` would be more helpful here.